pandas/core/frame.py

```diff
@@ -613,8 +613,52 @@ def __init__(
                     values, index, columns, dtype=values.dtype, copy=False
                 )
 
+        manager = get_option("mode.data_manager")
```

Comment:
No, because you need to be able to change the setting after initial import. 
Tom asked about performance impact of this above (https://github.com/pandas-dev/pandas/pull/36010#discussion_r492670548), and I did some timings then. It doesn't seem to be a problem.